dev/loadgen/configs/prod-benchmark-pvc.yaml

```diff
@@ -14,11 +14,14 @@ featureFlags:
 # https://github.com/gitpod-io/gitpod/blob/e437e1868072dfad1d195031a4709a254bd60dc8/components/ws-manager-api/core.proto#L596
 # from core.proto: PERSISTENT_VOLUME_CLAIM = 7;
   - 7
+# repoAuth:
+#   authUser: ""
+#   authPassword: ""
 repos:
   - cloneURL: https://github.com/Furisto/workspace-stress
     cloneTarget: main
     score: 20
-    workspaceImage: eu.gcr.io/gitpod-dev/dev/loadtest/workspace-stress:latest
+    workspaceImage: registry.hub.docker.com/gitpod/workspace-full:latest
```

Comment:
What's the motivation to switch to Docker Hub? Also, I think this image should be a reference to something that was built by image builder, not dazzle. Looping in @aledbf (below) for posterity.